google-cloud-examples/src/main/java/com/google/cloud/examples/storage/snippets/StorageSnippets.java

```diff
@@ -718,6 +718,19 @@ public Acl getBucketAcl(String bucketName) {
   }
 
   /**
+   * Example of getting the ACL entry for a specific user on a bucket.
+   */
+  // [TARGET getAcl(String, Entity)]
+  // [VARIABLE "my_unique_bucket"]
+  // [VARIABLE "google-cloud-java-tests@java-docs-samples-tests.iam.gserviceaccount.com"]
+  public Acl getBucketAcl(String bucketName, String userEmail) {
+    // [START storagePrintBucketAclForUser]
```

Comment:
Yeah - there's a [nascent effort](go/veneer-docs-samples) to start standardizing on region tag names across languages, and that's what they wanted for this sample. Why? What seems odd about it?